server/models/procedures.sql

```diff
@@ -1334,6 +1334,20 @@ BEGIN
   ON DUPLICATE KEY UPDATE `quantity` = `quantity` + movementQuantity;
 END $$
 
+-- stock movement document reference
```

Comment:
You should add more comment to help other understand what is done in this stored procedure

Reply:
👍